keps/sig-node/2371-cri-pod-container-stats/README.md

```diff
@@ -286,8 +286,7 @@ as cAdvisor is fine tuned to perform in an adequate manner.
 ### Stats Summary API
 
 #### CRI Implementation
-The CRI implementation will need to be extended to support reporting the full set of container-level from the [Summary API](#summary-container-stats-object).
-
+The CRI implementation will need to be extended to support reporting the full set of container-level from the [Summary API](#summary-container-stats-object). A new gRPC call will also be added to the CRI that allows reporting for metrics currently exported by cAdvisor, but are outside the scope of the Summary API. This new gRPC call will return a Prometheus metric based response which Kubelet can export. Additionally, PodAndContainerStatsFromCRI feature gate support will be added to only report Prometheus based metrics from the CRI when calling /metrics/cadvisor endpoint when the feature gate is enabled. The additional metrics we support will need to be added to the individual container runtimes.
```

Comment:
sorry, one more nit: we surround `PodAndContainerStatsFromCRI` and `/metrics/cadvisor` with backticks ` here and below

Reply:
Gotcha, updated

Reply:
/retest